.github/workflows/ci.yml

```diff
@@ -19,32 +17,40 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ "ubuntu-latest", "macos-latest"]
-        python-version: [ "3.7", "3.8", "3.9", "3.10" ]
+        os: [ "ubuntu-latest", "macos-latest", "macos-14" ]
+        python-version: [ "3.9", "3.10", "3.11" ]
 
     steps:
       - name: Cancel previous runs
-        uses: styfle/cancel-workflow-action@0.9.1
+        uses: styfle/cancel-workflow-action@0.11.0
         with:
           access_token: ${{ github.token }}
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           token: ${{ github.token }}
-      - name: Conda setup
+      - name: Conda setup (Linux)
         uses: conda-incubator/setup-miniconda@v2
+        if: matrix.os == 'ubuntu-latest' 
         with:
-          activate-environment: wrf_python_build
           python-version: ${{ matrix.python-version }}
-          channels: conda-forge, ncar
-      - name: Conda install (Darwin)
+          channels: conda-forge
+          environment-file: build_envs/Linux.yml
+      - name: Conda setup (macOS)
```

Comment:
We can also consider renaming its env file from "Darwin" to "MacOS.yml"

Reply:
Yeah, agree that would be more clear.  

I was hoping to skip that change and just move to the universal env file (see below), but I'll make the change since we'll have these at least a bit longer.